snakemake-slurm.md

```diff
@@ -0,0 +1,100 @@
+Snakemake via SLURM
+====
+
+Snakemake can submit each of your jobs to the slurm scheduler for you! To enable this, you need to provide the `--cluster` option to snakemake on the command line, and include all of the `sbatch` information you normally put at the top of your submission files.
+
+
+```
+snakemake --cluster "sbatch -A CLUSTER_ACCOUNT -t CLUSTER_TIME -p CLUSTER_PARTITION -N CLUSTER_NODES -J JOBNAME" —jobs NUM_JOBS_TO_SUBMIT
+```
+Notes: 
+  - Most clusters would prefer if you run use an interactive session to run this, so that you're not running anything on the login nodes. Since this process is only submitting jobs, you _can_ run this command on tmux/screen on a login node, but only do it for a small number of jobs (otherwise you'll slow everyone up and your job will be killed by admin!).
```

Comment:
Most clusters would prefer if you use an interactive session to run this, so that you're not running anything on the login nodes. Since this process is only submitting jobs, you _can_ run this command on tmux/screen on a login node, but only do it for a small number of jobs (otherwise you'll slow everyone up and your job will be killed by admin!).
```suggestion
  - Most clusters would prefer if you run use an interactive session to run this, so that you're not running anything on the login nodes. Since this process is only submitting jobs, you _can_ run this command on tmux/screen on a login node, but only do it for a small number of jobs (otherwise you'll slow everyone up and your job will be killed by admin!).
```